src/pages/register.js

```diff
@@ -46,12 +46,184 @@ export default () => (
+                    >
+                      <option>Undecided</option>
+                      <option>Computer Science</option>
+                      <option>Computer Engineering</option>
+                      <option>Information Technology</option>
+                      <option>Other</option>
+                    </select>
+                    <input
+                      className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                      name="major"
+                      type="text"
+                      placeholder="Other major"
+                      disabled
+                    />
+                  </div>
+                  <div className="mb-4 text-left">
+                    <label
+                      className="block text-gray-700 text-sm font-bold mb-2"
+                      htmlFor="graduation"
+                    >
+                      Expected Graduation Year
+                    </label>
+                    <input
+                      className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                      name="graduation"
+                      type="text"
+                      placeholder="2023"
+                      maxLength="4"
+                    />                    
+                  </div>
+                  <div className="mb-4 text-left">
+                    <label
+                      className="block text-gray-700 text-sm font-bold mb-2"
+                      htmlFor="hackathonsAttended"
+                    >
+                      Number of Hackathons attended
+              
+                    </label>
+                    <select
+                      className="shadow appearance-none border rounded py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                    >
+                      <option>0</option>
+                      <option>1</option>
+                      <option>2</option>
+                      <option>3</option>
+                      <option>4</option>
+                      <option>4+</option>
+                    </select>
+                  </div>
+                  <div className="mb-4 text-left">
+                    <label
+                      className="block text-gray-700 text-sm font-bold mb-2"
+                      htmlFor="resumeURL"
+                    >
+                      Upload resume (optional)
+                    </label>
+                    <input
+                      className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                      name="resumeURL"
+                      type="file"
+                    />
+                    </div>
+                  <div className="mb-4 text-left">
```

Comment:
- Add `w-full` to be consistent
- Change default to "Select an option"
- Remove "(optional)" from the label
- Add "Prefer not to answer" as an option